### PR TITLE
Updating Clang Version to 11.0.0

### PR DESCRIPTION
**Pull request by codereport**
Follow up PR to: https://github.com/rapidsai/cudf/pull/6695. Performing the same changes but for `rapidsai/cugraph`

Depends on: https://github.com/rapidsai/integration/pull/304

### CODE REVIEW

File: ci/checks/style.sh [hunk 1]
```diff
@@ -20,6 +20,10 @@ PATH=/conda/bin:$PATH
 # Activate common conda env
 source activate gdf
 
+gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
+gpuci_conda_retry install -y "clang=11.0.0"
+gpuci_conda_retry install -y "clang-tools=11.0.0"
```
Comment (dillon-cullinan) [inline ci/checks/style.sh:25]: These lines can be removed. Once the version is updated in integration, our images will come with the correct package.

**Review by dillon-cullinan — CHANGES_REQUESTED** [review]

File: ci/checks/style.sh (hunk 1 above)
Comment (codereport) [reply ci/checks/style.sh]: Done!

**Review by dillon-cullinan — APPROVED** [review]

**Review by ChuckHastings — APPROVED** [review]

**Comment by BradReesWork** [comment]
@codereport there are some style issues that need to be fixed

**Comment by ChuckHastings** [comment]
The fundamental issue with the style checker is that a number of files were updated for format (the clang format rules changed) but the copyright dates weren't updated to reflect that the contents of the files were changed.

**Comment by codereport** [comment]
rerun tests

**Comment by codereport** [comment]
rerun tests

File: cpp/src/experimental/induced_subgraph.cu [hunk 2]
```diff
@@ -199,14 +199,19 @@ extract_induced_subgraphs(
...
             thrust::seq, subgraph_offsets, subgraph_offsets + num_subgraphs, size_t{i}));
-        vertex_t const *indices{nullptr};
-        thrust::optional<weight_t const *> weights{thrust::nullopt};
+<<<<<<< HEAD
+        vertex_t const* indices{nullptr};
+        weight_t const* weights{nullptr};
+=======
+        vertex_t const* indices{nullptr};
+        thrust::optional<weight_t const*> weights{thrust::nullopt};
+>>>>>>> branch-21.08
```
Comment (ChuckHastings) [inline cpp/src/experimental/induced_subgraph.cu:214]: Looks like an improperly merged change here

Comment (codereport) [reply cpp/src/experimental/induced_subgraph.cu]: Hopefully fixed!

**Comment by codereport** [comment]
@gpucibot merge

**Comment by ajschmidt8** [comment]
@gpucibot merge

**Pull request merged**